Source/JavaScriptCore/runtime/ArrayConstructor.cpp

```diff
@@ -108,7 +110,13 @@ static ALWAYS_INLINE bool isArraySlowInline(JSGlobalObject* globalObject, ProxyO
 
     while (true) {
         if (proxy->isRevoked()) {
-            throwTypeError(globalObject, scope, "Array.isArray cannot be called on a Proxy that has been revoked"_s);
+            auto* callFrame = vm.topJSCallFrame();
+            auto* callee = callFrame ? callFrame->jsCallee() : nullptr;
+            String calleeName = "Array.isArray"_s;
+            auto* fn = jsDynamicCast<JSFunction*>(callee);
+            if (fn->intrinsic() == ObjectToStringIntrinsic)
```

Comment:
There is no guarantee that `callee` can be `JSFunction`. So let's do null check.
And we would like to use longer variable name `function` instead of `fn`.

Reply:
I've done this